tools/rw-heatmaps/README.md

```diff
@@ -12,15 +12,11 @@ To get a mixed read/write performance evaluation result:
 ```
 `rw-benchmark.sh` will automatically use the etcd binary compiled under `etcd/bin/` directory.
 
-Note: the result csv file will be saved to current working directory. The working directory is where etcd database is saved. The working directory is designed for scenarios where a different mounted disk is preferred.
+Note: the result CSV file will be saved to current working directory. The working directory is where etcd database is saved. The working directory is designed for scenarios where a different mounted disk is preferred.
 
 ### Plot Graphs
-To generate two images (read and write) based on the benchmark result csv file:
+To generate two images (read and write) based on the benchmark result CSV file:
 ```sh
 # to generate a pair of read & write images from one data csv file
-./plot_data.py ${FIRST_CSV_FILE} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
-
-
-# to generate a pair of read & write images by comparing two data csv files
-./plot_data.py ${FIRST_CSV_FILE} ${SECOND_CSV_FILE} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
+go run . ${CSV_FILE} -t ${IMAGE_TITLE} -o ${OUTPUT_IMAGE_NAME}
```

Comment:
Based on the help output it doesn't look like the short flags are in place yet?

```bash
  -h, --help                       help for rw-heatmaps
      --output-format string       output image file format (default "png")
      --output-image-file string   output image filename (required)
      --title string               plot graph title (required)

```

Reply:
Right, I forgot to declare them :facepalm: